Steam Web Integration.user.js

```diff
@@ -28,7 +28,7 @@
 // @run-at       document-start
 // @supportURL   https://github.com/Revadike/SteamWebIntegration/issues/
 // @updateURL    https://github.com/Revadike/SteamWebIntegration/raw/master/Steam%20Web%20Integration.user.js
-// @version      1.9.8
+// @version      1.9.10

```

Comment:
If it's not a hotfix, but multi-feature update, it should increment the 2nd number. Thus, I'd change it to `1.10.0`.

Reply:
Also, feel free to add yourself as `@contributor` in the metablock (on line 10)

Reply:
Then I will modify the version directly.

Reply:
Okay